tests/test_ext_marshmallow.py

```diff
@@ -684,6 +683,46 @@ def test_schema_partially_v3(self, spec_fixture):
             },
         }
 
+    def test_parameter_reference(self, spec_fixture):
+        if spec_fixture.spec.openapi_version.major < 3:
+            param = {"schema": PetSchema}
+            reference = '#/parameters/Pet'
```

Comment:
Note: I'd like to extend `tests.utils.ref_path` to parameter, responses and all kinds of components (https://github.com/marshmallow-code/apispec/pull/408#issuecomment-478176641). This can be done later.